### DISCUSSION TITLE
Transfer operators for azure blob storage

### DISCUSSION THREAD

**Discussion by chinwobble** [apache/airflow#25517] [Ideas]
### Description

Create Microsoft Azure Transfer Operators to Azure Blob Storage.

This could include:
Azure Table Storage to blob storage
Azure SQL Database to blob storage 
Azure Cosmos DB to blob storage
salesforce to blob storage

### Use case/motivation

Make it easier to do data ingestion for azure data services to blob storage / azure data lake.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Feel free!
